src/sources/prometheus/mod.rs

```diff
@@ -0,0 +1,204 @@
+use crate::{topology::config::GlobalOptions, Event};
+use futures::{sync::mpsc, Future, Sink, Stream};
+use http::Uri;
+use hyper;
+use hyper_tls::HttpsConnector;
+use serde::{Deserialize, Serialize};
+use std::time::{Duration, Instant};
+use tokio::timer::Interval;
+
+pub mod parser;
+
+#[derive(Deserialize, Serialize, Clone, Debug)]
+struct PrometheusConfig {
+    host: String,
```

Comment:
Have you thought about how we could support scraping multiple hosts from a single source?

Reply:
Right, forgot about this requirement! Fixed now.